module2.js

```diff
@@ -110,8 +123,9 @@ function question6(promise) {
 */
 
 function question7() {
-  return fetch('https://example.com')
-  // TODO
+  return fetch("https://example.com").then(function(response) {
+    return response.text();
```

Comment:
When you run the test, you get an error saying the `text()` method is undefined. Not sure it works 100%?